tests/other/metadce/hello_libcxx_O2_fexceptions.size

```diff
@@ -1 +1 @@
-165863
+176401
```

Comment:
Yikes, this is pretty significant... is it really from that one tiny change in library.js?